test/spix/smoke_test.py

```diff
@@ -38,41 +58,36 @@ def app():
         "./output/bin/qfield", stdout=subprocess.PIPE, stderr=subprocess.PIPE
     )
     app = xmlrpc.client.ServerProxy("http://localhost:9000")
+    app.process = process
 
     start = time.time()
     while True:
         try:
             exit_code = process.poll()
             if exit_code is not None:
-                output, errors = process.communicate()
-                print("== OUTPUT ==")
-                print(output.decode("utf-8"))
-                print("== ERRORS ==")
-                print(errors.decode("utf-8"))
-
+                process_communicate(process)
                 assert exit_code is None
             app.existsAndVisible("mainWindow")
             yield app
             break
         except (ConnectionRefusedError, OSError) as e:
             if time.time() - start > 30:
-                output, errors = process.communicate()
-                print(output)
-                print(errors)
+                process_communicate(process)
                 assert False  # Could not start app after 30 seconds
             print(str(e))
             time.sleep(0.2)
         except Exception as e:
-            output, errors = process.communicate()
+            process_communicate(process)
             assert False  # Unexcpected exception while starting up app
 
     app.quit()
     app.quit()
     try:
-        process.wait(1)
+        process.wait(5)
     except subprocess.TimeoutExpired:
-        print("Process did not quit after 1 second. Killing it.")
+        print("Process did not quit after 5 second. Killing it.")
```

Comment:
```suggestion
        print("Process did not quit after 5 second(s). Killing it.")
```
Also having "5" as a variable `WAIT_BEFORE_FORCE_QUIT_S` would be great.

Reply:
Already done locally